.azure-pipelines.yml

```diff
@@ -160,47 +163,54 @@ jobs:
       displayName: Set tag name to "master"
       condition: eq(variables['Build.SourceBranch'], 'refs/heads/master')
```

Comment:
I just realized that this doesn't cover pull request builds at all—that's the cause of the wrong file names I [mentioned before](https://github.com/CraneStation/wasmtime/pull/244#issuecomment-518409471).

Perhaps we should include something like `pull-request-#` in the name instead? There's `system.pullRequest.pullRequestNumber`, which could be used for that.

Reply:
I like the name "dev" from https://github.com/CraneStation/wasmtime/pull/242 and I don't think the PR artifacts will be consumed all that often because only members of the azure project can download them, so I'm gonna negate the tag condition and just name everything "dev" that isn't a tag

Reply:
I guess that's a good point, yes. I was wondering about potentially making these publicly available, too, but at least for now we should probably not do that.

Reply:
I was poking around and sort of wished that we could make the artifacts on azure pipelines public in case contributors were interested, but I'm not sure there's even a way to do so :(

I think in any case though having releases always live on the master branch will be a great step forward